apps/server/src/modules/deletion/uc/builder/deletion-request-body-props.builder.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { DeletionDomainModel } from '../../domain/types';
+import { DeletionRequestBodyPropsBuilder } from './deletion-request-body-props.builder';
+
+describe(DeletionRequestBodyPropsBuilder.name, () => {
+	afterAll(() => {
+		jest.clearAllMocks();
+	});
+	describe('when create deletionRequestBodyParams', () => {
+		it('should build deletionRequestBodyParams with all attributes', () => {
+			// Arrange
+			const domain = DeletionDomainModel.PSEUDONYMS;
```

Comment:
Use const setup placed behind the when description and use for refId = new ObjectId().toHexString();

Reply:
done